subprojects/packagefiles/quazip/meson.build

```diff
@@ -1,12 +1,18 @@
 project(
   'QuaZip',
   ['c', 'cpp'],
-  version: '0.7.3',
+  version: '1.3',
+  meson_version: '>=0.59.0',
   license: 'LGPL-2.1 with static linking exception'
 )
 
-qt5 = import('qt5')
-qt5_dep = dependency('qt5', modules: 'Core')
+qt_dep = dependency('qt5', modules: 'Core', required : false)
+if qt_dep.found()
+  qt = import('qt5')
+else
+  qt = import('qt6')
+  qt_dep = dependency('qt6', modules: 'Core')
+endif
```

Comment:
Interestingly, upstream has a build variable (not option, remember, "what are options") to select the version of Qt to use, and if unspecified it will go look for all 3 in the order: qt6, qt5, qt4